### DISCUSSION TITLE
CSS not being applied in pages

### DISCUSSION THREAD

**Discussion by johnray214** [community/community#113636] [Discussions]
### Select Topic Area

Question

### Body

Hello im new to github and i try to upload my newly created website here but my css is not working how can i fix this https://github.com/johnray214/sampleLab.github.io

**Comment by dizzpy** [comment]
i seems it's working properly !!
what is the issue ??

**Reply by johnray214** [reply]
yah i fix it

**Comment by Zain-Muhammad** [comment]
Hi, i tried to clone your repo and it works perfectly for me
![image](https://github.com/community/community/assets/68952585/28d25ed5-3262-4333-9625-ec80eec18912)
I check by cloning your file it's working fine but your github page URL showing this.
![image](https://github.com/community/community/assets/68952585/fc82c58a-5c63-4da1-a0f1-95e51843e982)
read the documentation and setup your page properly i am sharing documentation link too if you need any other assitance then let me know.
[](https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site)

**Reply by johnray214** [reply]
thanks but i think its ok now

**Reply by johnray214** [reply]
the url you check is wrong this is my link https://johnray214.github.io/sampleLab.github.io/

**Reply by Zain-Muhammad** [reply]
Ok Nice to hear! can you share your hosted link page url?

**Reply by johnray214** [reply]
yah the link that i comment is the hosted link you can visit it

**Reply by Zain-Muhammad** [reply]
my bad yes it's working fine now.
![image](https://github.com/community/community/assets/68952585/96c77ac7-05c2-4d59-9321-77b508bc01a4)
